frontend/packages/console-plugin-sdk/src/typings/storage-class-params.ts

```diff
@@ -0,0 +1,35 @@
+import { Extension } from '@console/plugin-sdk';
+
+namespace ExtensionProperties {
+  export interface StorageClassProvisioner {
+    getStorageClassProvisioner: () => ExtensionSCProvisionerProp;
```

Comment:
any reason to have this as function and not just 
```suggestion
    storageClassProvisioner: ExtensionSCProvisionerProp;
```
?